common/tags/pom.xml

```diff
@@ -34,6 +34,14 @@
     <sbt.project.name>tags</sbt.project.name>
   </properties>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.scalatest</groupId>
+      <artifactId>scalatest_${scala.binary.version}</artifactId>
+      <scope>compile</scope>
+    </dependency>
+  </dependencies>
```

Comment:
So far, I used both mvn and sbt to do a full rebuild in the command line. I also click `Invalidate Caches/Restart` in IntelliJ. I tried shutdown zinc. Still got the same error. 

To bypass/resolve the compilation failure, I have to add the above dependency back in my local environement. After this, everything works well in IntelliJ.